src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -246,6 +251,11 @@ private void setup(OptionSet options, Properties properties) {
 		if ( options != null && options.has("replay")) {
 			this.replayMode = true;
 		}
+
+		boolean outputIncludesBinlogPosition = options != null && options.has("output_includes_binlog_position");
+		boolean outputIncludesCommit = options != null && options.has("output_includes_commit");
+		boolean outputIncludesXid = options != null && options.has("output_includes_xid");
```

Comment:
gonna copy https://github.com/zendesk/maxwell/pull/420/files#diff-d24312bad9cc0cd2c507665e889b2c1aR146 so we don't end with two implementations of the same thing